drivers/display/display_st7789v.c

```diff
@@ -413,6 +434,10 @@ static const struct st7789v_config st7789v_config = {
 #if DT_INST_NODE_HAS_PROP(0, reset_gpios)
 	.reset_gpio = GPIO_DT_SPEC_INST_GET(0, reset_gpios),
 #endif
+#if DT_INST_NODE_HAS_PROP(0, backlight_led)
+	.backlight_dev = DEVICE_DT_GET_OR_NULL(DT_PARENT(DT_INST_PHANDLE(0, backlight_led))),
+	.backlight_led = DT_NODE_CHILD_IDX(DT_INST_PHANDLE(0, backlight_led)),
```

Comment:
I will leave this change to @Mani-Sadhasivam to ACK, because I'm not sure it's correct. It seems like this might be a little bit hard-coded to the gpio-leds compatible. Are there any other types of LED in devicetree that you might need to handle differently?

Reply:
I think this is fine IMO. Only the `gpio-leds` are special as they are referenced by different applications like LCD, Keyboard etc... for backlight purposes. The LED controllers mostly have a dedicated application controlling them as a whole so the knowledge of the pattern is obvious.

Moreover, the usage of a dedicated property allows us to follow what Linux has been doing for decades.